packages/core/src/App/Constants/platform-config.js

```diff
@@ -17,20 +17,6 @@ const platform_config = [
         description: () => localize('Automated trading at your fingertips. No coding needed.'),
         link_to: routes.bot,
     },
-    {
```

Comment:
@thisyahlen-deriv I just remembered we need these pages for mt5 and derivx changing password redirection, please DO NOT REMOVE this routes

Reply:
Will re-add them back